### ISSUE TITLE
[TF Lite] How to use XNNPACK delegate on Windows?

### ISSUE THREAD

**Issue by DwayneDuane** [tensorflow/tensorflow#47007]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): Source
- TensorFlow version: 2.5 (nightly)
- Python version: 3.8
- Installed using virtualenv? pip? conda?: No
- Bazel version (if compiling from source): 3.7.2
- GCC/Compiler version (if compiling from source): Visual C++ 2019
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the problem**
I was successfully building tensorflow lite on Windows, for CPU only, not with any delegate. I also verified that I was able to against `tensorflowlite_c.dll.if.lib` and to execute a tf lite model and produce the correct outputs.

Then I rebuilt tensorflow lite with the `xnnpack` delegate. The build was successful. However, now, when I try to build my test program and link against `tensorflowlite_c.dll.if.lib`, I now get:

![err1](https://user-images.githubusercontent.com/67419721/107218568-c80e4780-69dd-11eb-89f2-cebdc47dad81.PNG)

I tried to add `xnnpack_delegate.lib` to the list of library dependencies, only to get more linker errors:

![err2](https://user-images.githubusercontent.com/67419721/107218646-e2482580-69dd-11eb-960a-9ac33144f1e2.PNG)

I also tried linking against of libs, like `c_api.lo.lib` or `common.lo.lib`, but it made no difference. 
Anybody know which `.lib` files should I be linking against on Windows?

**Provide the exact sequence of commands / steps that you executed before running into the problem**

The command I used to build TF Lite with xnnpack delegate was:
```
bazel build -c opt --cxxopt=-std=c++14 --define tflite_with_xnnpack=true //tensorflow/lite/c:tensorflowlite_c.dll
```
When I ran the `configure.py` script, I left everything to default, with the only exception that I changed the `-c opt` flag from default `/arch:AVX` to `/arch:AVX512`.

**Comment by miaout17** [comment]
@multiverse-tf Would you be able to answer the XNNPack question or could you find someone who can answer? Thanks

**Comment by db81** [comment]
You get unresolved symbols because `TfLiteXNNPack*()` stuff doesn't get exported into the DLL on Windows. See #51882 for a fix, you can safely apply it against 2.6.0.
